Content.Shared/Hands/EntitySystems/SharedHandsSystem.Pickup.cs

```diff
@@ -103,6 +103,15 @@ protected virtual void HandleEntityInserted(EntityUid uid, HandsComponent hands,
         if (!CanPickupToHand(uid, entity, hand, checkActionBlocker, handsComp, item))
             return false;
 
+        var gotAttempt = new GotAttemptedHandPickupEvent(uid, entity, hand);
+        var didAttempt = new DidAttemptedHandPickupEvent(uid, entity, hand);
+
+        RaiseLocalEvent(entity, gotAttempt, false);
+        RaiseLocalEvent(uid, didAttempt, false);
+
+        if (gotAttempt.Cancelled || didAttempt.Cancelled)
+            return false;
+
```

Comment:
This should absolutely be a part of "CanPickupToHand" as otherwise this code will get ignored entirely for the other pickup methods.

Additionally are you sure the container insertion messages don't already cover this off? Also broadcast is already default to false so there's no reason to re-specify it either.

Reply:
Last I heard you were taking a break so these comments are more for Slam so no need to worry about responding if your not up for it right now!

I see what your saying, but it actually won't work with that :(
The issue, is that a lot of other places are using `CanPickupToHand` in bad ways (Try right clicking a lubed item, it'll throw it even though you never actually picked it up because its being used in the verb menu). I was told in the past to just make a new event for this but if you guys think I should using `CanPickupToHand` I'll give it a shot.